<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <widget class="QPushButton" name="pushButton_select_file">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>36</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTreeView" name="list_queryresult">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>781</width>
     <height>391</height>
    </rect>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_outputfile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>306</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>c:\temp\test.xls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open_file">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
